index.d.ts

```diff
@@ -33,9 +33,12 @@ declare module 'io-react-native-zendesk' {
   // function to shows all the tickets of the user
   export function showTickets(): void;
 
-  // function the return the number of tickets created by the user
+  // function that return the number of tickets created by the user
   export function hasOpenedTickets(): Promise<number>;
 
+  // function that return the number of unread messages by the user
+  export function hasNewResponse(): Promise<number>;
```

Comment:
```suggestion
  export function hasNewResponses(): Promise<number>;
```
Using the singular the name could suggest true/false value

Reply:
You're right, thank you! Modified in [8de728a](https://github.com/pagopa/io-react-native-zendesk/pull/17/commits/8de728a3b28f23ddead68d2ec43e5275797d6b32)